README.md

```diff
@@ -1 +1,23 @@
-# helloMicroverse
\ No newline at end of file
+# Project Name:hello microverse
```

Comment:
- [ ] The Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), as for now `feature` is not an action or feature, kindly in the future make sure you name your branch accordingly